README.md

```diff
@@ -1,41 +1,138 @@
+| ------------- | ------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| loadReference | `"false"`     | Does not do anything, however, is supported per the [Okapi specification](https://github.com/folio-org/okapi/blob/master/doc/guide.md)                                                                                                                                                                                        |
+| loadSample    | `"false"`     | If `"true"`, a series of sample calendars will be added representing examples of exceptions and openings across multiple calendars and service points. This will use the same service points loaded by [mod-inventory-storage](https://github.com/folio-org/mod-inventory-storage/tree/master/reference-data/service-points). |
 
-See the built `target/ModuleDescriptor.json` for the interfaces that this module
-requires and provides, the permissions, and the additional module metadata.
+### Environment variables
 
-### API documentation
+See `.env.sample` for example values.
+
+| Name            | Default Value              | Description                                                                           |
+| --------------- | -------------------------- | ------------------------------------------------------------------------------------- |
+| JAVA_OPTIONS    | `XX:MaxRAMPercentage=66.0` | Sets `java` command-line options. Default is to set the maximum ram percentage to 66% |
+| DB_HOST         | `localhost`                | Postgres hostname                                                                     |
+| DB_PORT         | `5432`                     | Postgres port                                                                         |
+| DB_USERNAME     | `folio_admin`              | Postgres username                                                                     |
+| DB_PASSWORD     | `folio_admin`              | Postgres password                                                                     |
+| DB_DATABASE     | `okapi_modules`            | Postgres database name                                                                |
+| DB_QUERYTIMEOUT | `60000`                    | Postgres query time out. Default is 6s                                                |
+| DB_CHARSET      | `UTF-8`                    | Postgres charset                                                                      |
+| DB_MAXPOOLSIZE  | `5`                        | Postgres max pool size                                                                |
```

Comment:
These three environment variables don't seem to be used anywhere in the code. 

The only references I can find are here, the module descriptor and the sample environment file.

Have I missed something?

Reply:
It's in Spring's configuration file: https://github.com/folio-org/mod-calendar/commit/6c4b8883a601c305ce46701f5a3b8bbf3219dac2#diff-b4c0421bcb3280a8e738550c830dba197057c515bcdd9579a52cbfe9adca2061R14

GitHub search is sometimes iffy when searching for something in interpolated strings

Reply:
@ncovercash That link doesn't really help me because it effectively takes me to the entire change (given these 25 000 changes were squashed to a single commit :-/ )

I didn't use the github search. I used IntelliJ with a local check out.

Which configuration file are you referring to? The only one I found was env.sample, which I would not consider as part of the code or an active configuration file.

Reply:
Spring's configuration file `src/main/resources/application.yaml` -- my apologies, I thought the link I gave went to a specific line.

Reply:
My apologies, I misread your original question.  It doesn't look like they are used and are just leftover from the previous mod-calendar; I'll remove them.